internal/wasm/jit/engine_test.go

```diff
@@ -454,20 +445,8 @@ func TestModuleEngine_Close_Concurrent(t *testing.T) {
 				mappedRegions = append(mappedRegions, e.compiledFunctions[f].codeSegment)
 			}
 
-			const goroutines = 100
```

Comment:
I will recreate something similar to this on store_test.go and that would make sure both interpreter and JIT only call once even if it is less important internally for interpreter. These tests will do things like ensure the first exit code is the one returned, even if called twice with different values.